### PR TITLE
Readme cleanup

### LINKED ISSUE #3242: The LinkPage link links to a spam site
It's not a link to the software itself but instead to a site with investment tips.

### LINKED ISSUE #3247: Relocated liteshort
[liteshort](https://github.com/132ikl/liteshort) has been relocated in [Gitea Instance](https://git.ikl.sh/132ikl/liteshort). Could anyone create PR?

### LINKED ISSUE #3260: [Want comments] Recomendation to remove apps.
1. [Taskfreak](https://www.taskfreak.com/original) - This project seems not to be active. The last change was 8 years ago. The community forum is closed.
1. [Taskord](https://gitlab.com/yo/taskord) - This project has been archived.

### PR DESCRIPTION
**Pull request by KieranRobson**
Closes #3242 
Closes #3247 
Closes #3260

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by mzch** [comment]
[liteshort](https://github.com/132ikl/liteshort) is now relocated to [author's Gitea instance](https://git.ikl.sh/132ikl/liteshort), so it must not be removed.

**Comment by KieranRobson** [comment]
> [liteshort](https://github.com/132ikl/liteshort) is now relocated to [author's Gitea instance](https://git.ikl.sh/132ikl/liteshort), so it must not be removed.

This has been updated in the latest commit

**Comment by mzch** [comment]
Oh, I missed it, Thanks @KieranRobson !

**Review by nodiscc — CHANGES_REQUESTED** [review]
Reviewed up to a4ce0aeec97b7739df678f6df21d1627c2ae0c1e

File: README.md [hunk 1]
```diff
@@ -593,7 +593,7 @@ _Related: [Conference Management](#conference-management)_
 
 - [BigBlueButton](https://bigbluebutton.org/) - Supports real-time sharing of audio, video, slides (with whiteboard controls), chat, and the screen. Instructors can engage remote students with polling, emojis, and breakout rooms. ([Demo](https://demo.bigbluebutton.org/gl), [Source Code](https://github.com/bigbluebutton/bigbluebutton)) `LGPL-3.0` `Java`
 - [Galene](https://galene.org/) - Galène (or Galene) is a videoconference server (an “SFU”) that is easy to deploy and that requires moderate server resources. ([Source Code](https://github.com/jech/galene)) `MIT` `Go`
-- [Jitsi Meet](https://jitsi.org/Projects/JitsiMeet) - Jitsi Meet is an OpenSource (MIT) WebRTC Javascript application that uses Jitsi Videobridge to provide high quality, scalable video conferences. ([Source Code](https://github.com/jitsi/jitsi-meet)) `MIT` `Javascript`
+- [Jitsi Meet](https://jitsi.org/Projects/JitsiMeet) - Jitsi Meet is an OpenSource (MIT) WebRTC Javascript application that uses Jitsi Videobridge to provide high quality, scalable video conferences. ([Source Code](https://github.com/jitsi/jitsi-meet)) `Apache-2.0` `Docker/Nodejs`
```
Comment (nodiscc) [inline README.md:596]: Docker is not a requirement, though a Docker-based install is possible. The main distribution method uses `.deb` packages. https://download.jitsi.org/stable/

Please change the language/platform tag to `Nodejs/Docker` to help clarify this.

File: README.md [hunk 2]
```diff
@@ -1217,7 +1217,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
...
 - [mStream](https://mstream.io/) - Music streaming server with GUI management tools. Runs on Mac, Windows, and Linux. ([Source Code](https://github.com/IrosTheBeggar/mStream)) `GPL-2.0` `Nodejs`
 - [musikcube](https://musikcube.com/) - Streaming audio server with Linux/macOS/Windows/Android clients. ([Source Code](https://github.com/clangen/musikcube)) `BSD-3-Clause` `C++`
-- [Navidrome Music Server](https://www.navidrome.org) - Modern Music Server and Streamer, compatible with Subsonic/Airsonic. ([Demo](https://www.navidrome.org/demo), [Source Code](https://github.com/navidrome/navidrome), [Clients](https://www.navidrome.org/docs/overview/#apps)) `GPL-3.0` `Go/Javascript`
+- [Navidrome Music Server](https://www.navidrome.org) - Modern Music Server and Streamer, compatible with Subsonic/Airsonic. ([Demo](https://www.navidrome.org/demo), [Source Code](https://github.com/navidrome/navidrome), [Clients](https://www.navidrome.org/docs/overview/#apps)) `GPL-3.0` `Docker/Go/Javascript`
```
Comment (nodiscc) [inline README.md:1220]: Javascript is not used server-side, please remove `/Javascript`

File: README.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ See [Contributing](.github/CONTRIBUTING.md).
 
 [Analytics](https://en.wikipedia.org/wiki/Analytics) is the systematic computational analysis of data or statistics. It is used for the discovery, interpretation, and communication of meaningful patterns in data.
 
-**Please visit [Awesome Analytics](https://github.com/newTendermint/awesome-analytics)**
+**Please visit [Awesome Analytics](https://github.com/0xnr/awesome-analytics)**
```
Comment (nodiscc) [inline README.md]: This commit (89c4d8e177a793db460c110a540986490f543021) accidentally reverts a bunch of changes. Can you please drop this commit?

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (nodiscc) [reply README.md]: 626400b86ca03c43cbacd16ada92ed8a25094cdb and following commits appear to fix the problem.

**Comment by KieranRobson** [comment]
Changes made!

File: README.md (hunk 3 above)
Comment (KieranRobson) [reply README.md]: Yeah haha, No idea why the merge reverted alot 😆

**Review by nodiscc — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1994,7 +1987,7 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 - [Blink](https://docs.blink.rest) - Easy-to-host, SSO-integrated, CDN-powered link shortener (+decoupled analytics) for teams. ([Source Code](https://github.com/JaneJeon/blink)) `AGPL-3.0` `Nodejs`
 - [goshorly](https://git.ucode.space/Phil/goshorly) - An easy self-hosted Link shortener in Golang with Redis <3. `MIT` `Go`
 - [Kutt](https://kutt.it) - A modern URL shortener with support for custom domains. ([Source Code](https://github.com/thedevs-network/kutt)) `MIT` `Nodejs`
-- [liteshort](https://github.com/132ikl/liteshort) - User-friendly, actually lightweight, and configurable URL shortener. `MIT` `Python`
+- [liteshort](hhttps://git.ikl.sh/132ikl/liteshort) - User-friendly, actually lightweight, and configurable URL shortener. `MIT` `Python`
```
Comment (Epy) [inline README.md:1990]: There is a little typo here `hhttps`

Comment (KieranRobson) [reply README.md:1990]: Fixed in more recent readme cleaup

**Pull request merged**